### ISSUE TITLE
The postgres backend for mdb-export uses double quotes for strings

### ISSUE THREAD

**Issue by turnovec** [mdbtools/mdbtools#119]
I tried to import an SQL dump created from Access database with `mdb-export -I postgres`. But when I tried to import this dump file into PostgreSQL, I've got many errors like this:
```
ERROR: column "ADMIN" does not exist
LINE 1: ..., "LOGIN", "PASSW") VALUES (1,"ADMIN","...
```
The first line of SQL dump file looks like this:
```
INSERT INTO "USER" ("USERID", "LOGIN", "PASSW") VALUES (1,"ADMIN","12345");
```
Then I've found that according to PostgreSQL specification, strings should not be enclosed with doublequotes (") but with singleqoutes ('). Would it be possible to correct this behaviour of mdb-export for this specific backend? Otherwise it is unusable...

**Comment by turnovec** [comment]
I found the -q parameter which I can use to set ' as quoting character for strings, but I still think that if I choose postgres as backend, it should be already included as default, because otherwise these exports are unusable for PostrgreSQL...

**Comment by shorabhd** [comment]
Thanks @turnovec 

That worked for me. But I agree it should be done by default.  @brianb Any thoughts?

**Comment by evanmiller** [comment]
Fix included in #162
